### PR TITLE
Markdown issue

### PR DESCRIPTION
**Pull request by Mbareck21**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,11 @@
     "@angular/ssr": "^17.3.7",
     "@ng-icons/bootstrap-icons": "^27.4.0",
     "@ng-icons/core": "^27.4.0",
+    "emoji-toolkit": "^8.0.0",
     "express": "^4.19.2",
+    "marked": "^12.0.2",
+    "ngx-markdown": "^18.0.0",
```
Comment (abdelhalimjean) [inline package.json:29]: use version 17.2.1

version 18.x.x is for Angular 18 and we're still on 17

**Review by abdelhalimjean — CHANGES_REQUESTED** [review]

**Comment by abdelhalimjean** [comment]
also fix the conflict @Mbareck21

File: package.json (hunk 1 above)
Comment (Mbareck21) [reply package.json]: Changed were made

**Review by Mbareck21 — COMMENTED** [review]
- Updated markdown version to v 17
- Fixed the merge conflicts

**Review by Mbareck21 — COMMENTED** [review]
Done

**Review by abdelhalimjean — APPROVED** [review]

**Pull request merged**